model/model.go

```diff
@@ -135,6 +135,16 @@ func (t *TableInfo) GetPkName() CIStr {
 	return CIStr{}
 }
 
+// GetPkColInfo gets the ColumnInfo of pk if exists.
+func (t *TableInfo) GetPkColInfo() *ColumnInfo {
+	for _, colInfo := range t.Columns {
+		if mysql.HasPriKeyFlag(colInfo.Flag) {
```

Comment:
If the logic is right, you can keep it untouched. But please add a comment here.